tests/vcom/vcom_test.go

```diff
@@ -0,0 +1,102 @@
+package vcom
+
+import (
+	"os"
+	"strings"
+	"testing"
+	"time"
+
+	tk "github.com/lf-edge/eden/pkg/evetestkit"
+	log "github.com/sirupsen/logrus"
+)
+
+var eveNode *tk.EveNode
+
+const (
+	sshPort = "8027"
+	appLink = "https://cloud-images.ubuntu.com/releases/22.04/release/ubuntu-22.04-server-cloudimg-amd64.img"
+)
+
+func TestMain(m *testing.M) {
+	log.Println("VCOM Test started")
+	defer log.Println("VCOM Test finished")
+
+	node, err := tk.InitilizeTest(m, "vcomlink")
+	if err != nil {
+		log.Fatalf("Failed to initialize test: %v", err)
+	}
+
+	eveNode = node
+	res := m.Run()
+	os.Exit(res)
+}
+
+func TestVcomLink(t *testing.T) {
+	log.Println("TestVcomLink started")
+	defer log.Println("TestVcomLink finished")
+
+	if !eveNode.IsTpmEnabled() {
+		t.Skip("TPM is enabled, skipping test")
+	}
+
+	t.Log("Checking if vcomlink is running on EVE")
+	stat, err := eveNode.EveRunCommand("eve exec pillar ss -l --vsock")
+	// vcomlink listens on port 2000 and host cid is 2.
+	// this is hacky way to check it is running, but it works ¯\_(ツ)_/¯
+	if !strings.Contains(string(stat), "2:2000") {
+		t.Fatalf("vcomlink is not running %v", err)
```

Comment:
This will likely print out `vcomlink is not running <nil>` when there is a problem with vsock. I think that printing `stat` instead of (or additionally to) `err` will be more useful.

Reply:
got catch!